File: vivid/src/main/java/com/icetea09/vivid/features/camera/CameraHelper.java
package com.icetea09.vivid.features.camera;

import android.content.Context;
import android.content.Intent;
import android.provider.MediaStore;

public class CameraHelper {
    public static boolean checkCameraAvailability(Context context) {
        Intent intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
        return intent.resolveActivity(context.getPackageManager()) != null;
    }
}

File: library/src/main/java/com/icetea09/vivid/camera/CameraHelper.kt
package com.icetea09.vivid.camera

import android.content.Context
import android.content.Intent
import android.provider.MediaStore

object CameraHelper {

    fun checkCameraAvailability(context: Context): Boolean {
        val intent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
        return intent.resolveActivity(context.packageManager) != null
    }

}
